.github/workflows/ci.yml

```diff
@@ -18,10 +23,20 @@ jobs:
     - uses: coursier/cache-action@v6
     - uses: laughedelic/coursier-setup@v1
       with:
-        jvm: 8
+        jvm: ${{ matrix.java }}
         apps: sbt-launcher
-    - name: Test
-      run: ./.github/scripts/ci.sh
+    - name: Test on java 8
```

Comment:
I wrote here according to `ci.sh`. Is it better to add  evictionCheck,mimaReportBinaryIssues, and so on for java 11?